lib/CarriageTheme.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:flutter/material.dart';
+
+class CarriageTheme {
+  static final TextStyle largeTitle = TextStyle(
+      color: Colors.black,
+      fontFamily: 'SFDisplay',
+      fontWeight: FontWeight.bold,
+      fontSize: 34,
+      letterSpacing: 0.37
+  );
+
+  static final TextStyle title1 = TextStyle(
+      color: Colors.black,
+      fontFamily: 'SFDisplay',
+      fontWeight: FontWeight.bold,
+      fontSize: 28,
+      letterSpacing: 0.36
+  );
+
+  static final TextStyle title2 = TextStyle(
+      color: Colors.black,
+      fontFamily: 'SFDisplay',
+      fontWeight: FontWeight.bold,
+      fontSize: 22,
+      letterSpacing: 0.35
+  );
+
+  static final TextStyle title3 = TextStyle(
+      color: Colors.black,
+      fontFamily: 'SFDisplay',
+      fontWeight: FontWeight.w600,
+      fontSize: 20,
+      height: 24 / 20,
+      letterSpacing: 0.38
+  );
+
+  static final TextStyle body = TextStyle(
+      color: Colors.black,
+      fontFamily: 'SFText',
+      fontWeight: FontWeight.w400,
+      fontSize: 17,
+      letterSpacing: -0.41
+  );
+
+  static final TextStyle button = TextStyle(
+      color: Colors.white,
+      fontFamily: 'SFText',
+      fontWeight: FontWeight.bold,
+      fontSize: 17
+  );
+
+  static final TextStyle subheadline = TextStyle(
+      color: Colors.black,
+      fontFamily: 'SFText',
+      fontWeight: FontWeight.w400,
+      fontSize: 15,
+      letterSpacing: -0.24
+  );
+
+  static final TextStyle caption1 = TextStyle(
+      color: Colors.black,
+      fontFamily: 'SFText',
+      fontWeight: FontWeight.w600,
+      fontSize: 11,
+      letterSpacing: 0.07
+  );
+
+  static final TextStyle caption2 = TextStyle(
+      color: Colors.black,
+      fontFamily: 'SFText',
+      fontWeight: FontWeight.w400,
+      fontSize: 11,
+      letterSpacing: 0.07
+  );
+
+  static Color mainBlack = Colors.black;
+  static Color gray1 = Color.fromRGBO(74, 74, 74, 1);
```

Comment:
Nit: For consistency, is there a way to stick with either hex or RGB? A lot of the colors elsewhere in this PR are hex while these are RGBO. Or is that a design decision?

Reply:
Whoops, this was just from me being lost on Figma. Will change to be consistent!

Reply:
Discussed offline, changed in a different commit